public/js/actions/pay-methods.js

```diff
@@ -5,6 +5,33 @@ import * as actionTypes from 'constants/action-types';
 import * as appActions from './app';
 
 
+export function delPayMethod(payMethodUri, jquery=$) {
+
+  if (typeof payMethodUri === 'undefined') {
+    throw new Error('payMethodUri is undefined');
```

Comment:
this will cause a 400 in the API response which might be better because in some theoretical future we'll have a user friendly catch all for 400 responses. But this works for now, just probably worth making future actions like this "dumb" for simplicity.